custom_components/deebot/sensor.py

```diff
@@ -181,3 +196,8 @@ def icon(self) -> Optional[str]:
             return "mdi:timer-outline"
         elif self._id == "stats_type":
             return "mdi:cog"
+
+    async def async_added_to_hass(self) -> None:
+        """Set up the event listeners now that hass is ready."""
+        listener = self._vacbot.statsEvents.subscribe(lambda _: self.schedule_update_ha_state())
+        self.async_on_remove(self._vacbot.cleanLogsEvents.unsubscribe(listener))
```

Comment:
> I added that all sensors are disabled by default and the user can enable only the sensor, which he needs.
> 
> During this change I recognized that we don't setup a device. This would be really helpful, as the user gets a overview of the device with it's corresponding entities, automation's, etc.
> See screenshot below from esphome:
> ![grafik](https://user-images.githubusercontent.com/26537646/110636643-cbe3e400-81ac-11eb-909b-976a08ba5427.png)
> 
> What do you think?

Yes i think it will be nice, i dont know if having sensor disable by default will confuse users, they might not see them

Reply:
User can see all sensors (deactivated or not) when they are clicking on the integration.
In the screenshot below for example I have disabled the first three sensors, as I don't need them

![grafik](https://user-images.githubusercontent.com/26537646/110651095-6ba86e80-81bb-11eb-8680-d0fb0377d394.png)

Reply:
i have no idea how to setup a device with entities in it, i couldnt find any documentation or example